### PR TITLE
[12.0][MIG] companyweb base

### PR DESCRIPTION
**Pull request by xavier-bouquiaux**
@sbidoul

### CODE REVIEW

**Comment by sbidoul** [comment]
Looks good. Could you squash the migration commits ?

**Review by sbidoul — APPROVED** [review]

File: companyweb_base/models/res_partner.py [hunk 1]
```diff
@@ -518,7 +518,7 @@ def cweb_button_enhance(self):
         )
         if r["StatusCode"] in [101, 302]:
             return self._cweb_call_wizard_credentials("Enter Companyweb credentials")
-        elif r["StatusCode"] != 0:
+        if r["StatusCode"] != 0:
```
Comment (rousseldenis) [inline companyweb_base/models/res_partner.py]: @xavier-bouquiaux Why changing elif here in the migration commit ?

**Review by rousseldenis — APPROVED** [review]

File: companyweb_base/models/res_partner.py (hunk 1 above)
Comment (sbidoul) [reply companyweb_base/models/res_partner.py]: I changed this to keep the code aligned across versions

**Pull request merged**